### PR TITLE
Added API route for Season services

### PR DESCRIPTION
**Pull request by fadyham**
Added API route for Season services

### CODE REVIEW

File: src/Models/IServiceInfo.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿/*
+ * Service Info
+ */
+export interface IServiceInfo {
+    id: number;
+    name: string;
+    order: number;
+    contentCount: ReadonlyStringMap<number>;
+}
```
Comment (behoyh) [inline src/Models/IServiceInfo.ts]: Needs ReadonlyStringMap<T> import statement

File: src/test/seasons.spec.ts [hunk 2]
```diff
@@ -80,4 +80,73 @@ describe("Seasons", () => {
...
+                .get("/seasons/badInput/services")
+                .end((err, res) => {
+                    res.should.have.status(404);
+                    res.body.should.be.a("object");
+                    res.body.should.have.property("message");
+                    done();
+                });
+        });
+
+        it("should return a 404 for negative season ids", (done) => {
```
Comment (behoyh) [inline src/test/seasons.spec.ts]: duplicate test

File: src/DataProviders/SqlDataProvider/SqlDataProvider.ts [hunk 3]
```diff
@@ -28,12 +31,37 @@ export class SqlDataProvider implements IDataProvider {
...
+        contentCount[FormatType.Audio] = serviceDbItem.Audio_Count;
+        contentCount[FormatType.Video] = serviceDbItem.Video_Count;
+        contentCount[FormatType.Information] = serviceDbItem.Information_Count;
+
+        return {
+            contentCount,
+            id: serviceDbItem.ItemId,
+            name: serviceDbItem.Structure_Name,
+            order: serviceDbItem.Service_Order
+        };
```
Comment (behoyh) [inline src/DataProviders/SqlDataProvider/SqlDataProvider.ts]: Perhaps a dedicated mapper class would be more appropriate here?

Comment (fadyham) [reply src/DataProviders/SqlDataProvider/SqlDataProvider.ts]: Not sure what you mean.  Example?

Comment (behoyh) [reply src/DataProviders/SqlDataProvider/SqlDataProvider.ts]: Another class such as GetServicesRepository.ts to call the SqlDataProvider.cs and provide a repository layer/mapping abstraction.

Comment (fadyham) [reply src/DataProviders/SqlDataProvider/SqlDataProvider.ts]: Not sure I follow.  The stored procedure already returns the counts for all the different formats

Comment (behoyh) [reply src/DataProviders/SqlDataProvider/SqlDataProvider.ts]: Maybe I'm the one who's not following 😁 LGTM

**Pull request merged**